apps/workshop/projects/Adax/helpers/queries.ts

```diff
@@ -0,0 +1,5 @@
+export async function accountAssets(axios: any, stakeAddress: string) {
```

Comment:
Why we use `any` here?

Reply:
we don't have access to Axios types from the workshop

Reply:
fixed